libs/ui-watt/src/lib/foundations/color/colors.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * @Please notice!!! 
+ * These values should be kept in-sync with _colors.import.scss
+ * Naming convention is as follow: --watt-{COLOR GROUP}-colors-{COLOR NAME}
+ */
+export enum Colors {
+    'primary' = '--watt-primary-colors-primary',
+    'primary-light' = '--watt-primary-colors-primary-light',
```

Comment:
Suggestion: Use Pascal case for enum keys, e.g. `PrimaryLight`. This will also make it unnecessary to quote the keys.

Suggestion: What do you think about the naming convention that looks like `--watt-color-primary`, `--watt-color-primary-dark`, `--watt-color-neutral-grey-100`, etc. instead? The `color` part after `--watt` can be used for grouping as well.

Reply:
Agreed, I will change that asap